content/en/docs/getting-started/linux-quickstart.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Try Falco on Linux
+description: Learn how to install Falco on Linux
+slug: falco-linux-quickstart
+aliases: [/falco-linux-quickstart]
+weight: 20
+---
+
+In this scenario you will learn how to install Falco on an Ubuntu 20.04 host, trigger a falco rule, and then examine the output. 
+
+The goal of this activity is to give you a quick example of how Falco works. After you complete it you should be able to move on to a more advanced [tutorial]( {{< ref "docs/tutorials" >}}) or spend some time reading up on [additional Falco concepts]( ../falco-additional ). 
+
+## Prerequisites
+This lab is based on installing Falco using the kernel module on Ubuntu. 
+
+The scenario has been tested specifically using Virutual Box and Lima for MacBooks running Apple Silicon. 
+
+While this tutorial may work with Ubuntu running on a cloud provider or another virtualization platform, it has not been tested. 
+
+### Virtual box setup 
+The following steps will set up a Virtual Box virtual machine running Ubuntu 20.04.
+
+* Install Virtual Box and Vagrant according to the instructions appropriate for your local system
+
+* Issue the following commands from the command line to create an Ubuntu 20.04 virtual machine
+
+```plain
+    vagrant init bento/ubuntu-20.04
+    vagrant up
+```
+
+* Log into the newly launched virtual machine and continue to the *Install Falco* section below (the default password is *vagrant*)
+
+```plain
+    ssh -p 2222 vagrant@127.0.0.1
+```
+
+### Lima setup for Apple silicon (M1/M2)
+This section explains how to create an Ubuntu 22.04 VM on Apple computers running M1 silicon (as opposed to Intel). 
+
+If you are unsure what processor your Apple machine is running, you can find out by clicking the Apple icon in the upper left, and choosing "About this Mac". The first item listed, Chip, tells you what silicon you're running on 
+
+* Install Brew according to the projects documentation
+
+* Use Brew to install Lima
+
+```plain
+    brew install lima
+```
+
+* Create an Ubuntu 20.04 VM with Lima
+
+```plain
+    limactl start --name=falco-quickstart template://ubuntu-lts
+```
+
+* Shell into the Ubuntu VM and once you're in the VM continue to the Install Falco section
+
+```plain
+    limactl shell falco-quickstart
+```
+
+## Install Falco
```

Comment:
We might want to say something to the effect of "regardless of which platform you used above..." to show that the process is the same from here on out for both platforms.